client/pages/quizzes.js

```diff
@@ -3,10 +3,10 @@
 */
 
 import React, { useState, useEffect } from "react";
-// import useSWR from "swr"; <-- uncomment when DB is ready
```

Comment:
Was there a reason to change these?
As I see it, it will revert to using the " once the comments are removed anyway

Reply:
Nah I think it's fine. I was just curious 🙂